src/Pages/ContentListTable/components/StatusIcon.tsx

```diff
@@ -22,13 +26,43 @@ const useStyles = createUseStyles({
     margin: '-3px 0',
   },
 });
+
 interface Props {
-  status?: string;
-  error?: string;
+  rowData: ContentItem;
+  retryHandler?: (string) => Promise<void>;
 }
 
-const StatusIcon = ({ status, error }: Props) => {
+const popoverDescription = (error?: string, count?: number, time?: string) => {
+  if (time === '') {
+    time = 'Never'
```

Comment:
In typescript/javscript in general, we try to avoid re-assigning props/imported variables.